### ISSUE TITLE
Map Quest: Uleguerand Range

### ISSUE THREAD

**Issue by panfunk** [EdenServer/community#640]
Cannot get the diary to show me page 4, to start the quest:

https://ffxiclopedia.fandom.com/wiki/Over_the_Hills_and_Far_Away?oldid=342648

Rank9 fame and did all prequests.

**Comment by bkesecker** [comment]
Partially blocked by https://github.com/EdenServer/community/issues/532

**Comment by bkesecker** [comment]
Got page 4 working in the Diary.  Need retail event IDs from Antreneau to kick off the quest.

**Comment by panfunk** [comment]
I still can't get it, client is up to date. 
Did you complete the 'Cure a cough' quest?

**Comment by bkesecker** [comment]
No, I am fixing the code so that it will work in the future.  Once I have it working completely, I'll issue a pull request into DarkStar Project.  It must be approved, however, and then Eden development can get it next time they sync with DSP master.

**Comment by bkesecker** [comment]
I should clarify.  To Cure A Cough quest will be flagged if the SQL provided in issue 532 is run on the DB.  This is something that can be done now on Eden.  I only tested flagging To Cure A Cough quest, not completing it.

**Comment by bkesecker** [comment]
I have the code finished for this one.  Will be making a pull request in DSP today.

**Comment by bkesecker** [comment]
https://github.com/DarkstarProject/darkstar/pull/5623

**Comment by bkesecker** [comment]
DSP pull request made.  https://github.com/DarkstarProject/darkstar/pull/5623

**Comment by bkesecker** [comment]
Pull request appears to be approved.

**Comment by bkesecker** [comment]
Pull request was merged to DSP master.

**Comment by panfunk** [comment]
Has this quest been implemented yet?
Asking cuz I still can't start it, fame/prequests done

**Comment by godmode** [comment]
It should be. Has anyone else been able to complete this quest?

**Comment by bkesecker** [comment]
It could be that one of your character's flags is in an invalid state.  What character name should we check?

**Comment by panfunk** [comment]
it's for both Pan & Chan

**Comment by bkesecker** [comment]
GM says your diary read page is at 2.  Please re-read the Diary and try again.

**Comment by bkesecker** [comment]
I see a scenario where reading the diary won't work.  If To Cure a Cough, The Medicine Woman, and A Squire's Test II are all in the completed state, you still won't be able to read pages 3 or 4.  I will fix this in a new PR.

**Comment by panfunk** [comment]
bingo

![image](https://user-images.githubusercontent.com/36666121/56470960-db3cd580-644c-11e9-9286-9873c28c06cf.png)

**Comment by bkesecker** [comment]
https://github.com/godmode/dsp/pull/630
